src/components/OpenFileDialog/OpenFileDialog.js

```diff
@@ -0,0 +1,444 @@
+                                                                            >
+                                                                                {getString(
+                                                                                    type,
+                                                                                    'newButtonLabel'
+                                                                                )}
+                                                                            </Button>
+                                                                        </div>
+                                                                    </>
+                                                                )}
+                                                            </div>
+                                                        </div>
+                                                    </Box>
+                                                </DataTableCell>
+                                            </DataTableRow>
+                                        )}
+                                    {Boolean(
+                                        data?.files[getResourceFromType(type)]
+                                            .length
+                                    ) && (
+                                        <FileList
+                                            type={type}
+                                            data={
+                                                data.files[
+                                                    getResourceFromType(type)
+                                                ]
+                                            }
+                                            onSelect={onFileSelect}
+                                        />
+                                    )}
+                                </DataTableBody>
+                            </DataTable>
+                            {Boolean(
+                                data?.files[getResourceFromType(type)].length
+                            ) && (
+                                <DataTableToolbar position="bottom">
+                                    <div className="pagination-controls">
+                                        <PaginationControls
+                                            page={page}
+                                            pager={data.files.pager}
+                                            onPageChange={setPage}
+                                        />
+                                    </div>
+                                </DataTableToolbar>
+                            )}
+                        </>
+                    )}
+                    <style jsx>{styles}</style>
+                </Box>
+            </ModalContent>
+        </Modal>
+    )
+}
+
+OpenFileDialog.defaultProps = {
+    open: false,
```

Comment:
`currentUser` missing a default prop or `isRequired`

Reply:
or, better yet, should be `isRequired`

Reply:
Required it is.